README.md

```diff
@@ -7,11 +7,18 @@ Docker file for creating mlab builder images.
 To build the docker image and tag it as m-lab/builder
 
     docker build -t m-lab/builder .
+OR
+    docker build github.com/m-lab/builder
```

Comment:
The error message `Cannot connect to the Docker daemon. Is the docker daemon running on this host?` might indicate that you need to run it with sudo.